tests/k6/src/test/benchmark/file-download.ts

```diff
@@ -0,0 +1,40 @@
+import {defaults, playbook} from '../../lib'
+import {Options} from 'k6/options';
+import {sleep} from "k6";
+import auth from "../../lib/auth";
+
+export const options: Options = {
+    ...defaults.K6.OPTIONS,
+};
+const authFactory = new auth(defaults.ACCOUNT.for(defaults.ACCOUNT.EINSTEIN));
+const plays = {
+    fileUpload: playbook.dav.fileUpload(),
+    fileDownload: playbook.dav.fileDownload(),
+    fileDelete: playbook.dav.fileDelete(),
+}
+export default () => {
+    const {login: userName} = authFactory.account;
+    const fileName = plays.fileUpload({
+        credential: authFactory.credential,
+        userName,
+        asset: defaults.FILE,
+    });
+
+    sleep(1)
+
+    plays.fileDownload({
+        credential: authFactory.credential,
+        userName,
+        fileName,
+    });
+
+    sleep(1)
+
+    plays.fileDelete({
+        credential: authFactory.credential,
+        userName,
+        fileName,
+    });
+
```

Comment:
@fschade looks like you have added uploadFile, deleteFile inside the default function. doesn't this make the metrics read from the download function a bit unreliable since when VUs are still downloading files some will have already downloaded and started to delete, That means the server is not only doing the downloading but also deleting at the same time.

Reply:
it does't, this is the reason why its build as a closure...
you can see at the top of the play that there are custom metrics. 

We get a detailed output metric for every action.

Reply:
we shouldnt care about the total execution time in most cases.
- its importand to identify how long did one interaction step take
- and if we care about total execution time we can integrate a metric that reflects every interaction flow per vu and iteration

Reply:
@fschade I totally get your point, but doesn't that make the metrics a bit unreliable, 
for eg. let's say we have a test for 100 UVs simultaneously uploading a file, 
In this case, I suppose we would want to measure the performance when the server is completely focusing its resources on handling the uploads, 
but in our case, we will be measuring when 100 VUs are simultaneously uploading, but also like 37 of them have also started deleting the files.
Sorry I might be misunderstanding something here.